dev/tests/functional/tests/MFTF/DevDocs/ActionGroup/DeprecatedCommentActionGroup.xml

```diff
@@ -8,6 +8,15 @@
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
     <actionGroup name="DeprecatedCommentActionGroup" deprecated="This Action Group is outdated and will be deleted next release.">
+        <arguments>
+            <argument name="test" type="string" />
+        </arguments>
         <comment userInput="Action group to demonstrate deprecation notices." stepKey="comment" />
+        <helper class="\MFTF\DevDocs\Helper\CustomHelper" method="goTo" stepKey="custobhhmHelper">
+            <argument name="test">{{contentSection.pageIntro}}</argument>
```

Comment:
Do arguments of custom helper support references to data entities?

Reply:
To anything

Reply:
can you give a usage example to reference an entity?

Reply:
Please see `HelperActionGroup`